### PR TITLE
ITEA-22-add-templates-from-itea-15-into-django-skeleton

### PR DESCRIPTION
**Pull request by RomanPryima**
Moved html's into properly folder.
Didn't make changes in settings, views, etc., files.
pictures are in folder "media".
Will investigate how to manage with background images.

### CODE REVIEW

**Comment by UkrainianEngineer** [comment]
@RomanPryima , Are you sure that it's ready for  review?

**Comment by RomanPryima** [comment]
@pivancy Now it is.

**Comment by UkrainianEngineer** [comment]
screenshot 2017-05-28 19 27 49
It will be better to have footer in the middle of the page.


@pivancy  corrected

**Comment by UkrainianEngineer** [comment]
Also, it should be some plugin, etc. instead of picture:
screenshot 2017-05-28 19 29 23

@pivancy Please explain what kind of plugin shell it be?

File: travel_platform/travel_app/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Query',
```
Comment (UkrainianEngineer) [inline travel_platform/travel_app/migrations/0001_initial.py:17]: Why this model is needed here?

Comment (RomanPryima) [reply travel_platform/travel_app/migrations/0001_initial.py:17]: @pivancy This file was created automatically during manage.py migrate command.
I know that there should be properly model, but have no idea, what should it be.

**Comment by UkrainianEngineer** [comment]
> @pivancy Please explain what kind of plugin shell it be?

It may be something from this:
http://techslides.com/50-javascript-libraries-and-plugins-for-maps

**Comment by UkrainianEngineer** [comment]
LGTM. 👍

**Comment by RomanPryima** [comment]
@kate-ka @nguyenduchuy @Sasha1152 please response on this pull request

**Comment by kate-ka** [comment]
👍

File: travel_platform/travel_app/templates/travel_app/main.html [hunk 2]
```diff
@@ -124,4 +122,19 @@
   </div>
 </div>
 
+<script>
```
Comment (UkrainianEngineer) [inline travel_platform/travel_app/templates/travel_app/main.html]: Btw, it's much better to have separate `*.js` files for javascript.

**Pull request merged**